src/common/applicationDetails/ApplicationDetails.tsx

```diff
@@ -214,6 +215,7 @@ const ApplicationDetails = ({
                 onConfirm={() => handleDeleteLease(lease.id)}
                 disabled={isDeletingLease}
                 buttonClassName={styles.deleteButton}
+                buttonStyle={ButtonWithConfirmationStyle.FLAT}
```

Comment:
I don't think this should need an imported enum for usage, should this be just `buttonStyle="flat"`?

Reply:
Done.